operator/deploy/operator-psp.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kiali-operator-psp
+  labels:
+    app: kiali-operator
+    version: v1.1.0
```

These files here are templates - do not hardcode the version. It will be set by the deploy script. So change this to the following:

```suggestion
    version: ${OPERATOR_VERSION_LABEL}
```